content/working_with/console/widgets/deploymentsView.md

```diff
@@ -0,0 +1,279 @@
+---
+layout: bt_wiki
+title: Deployments View
+category: Widgets
+draft: false
+---
+
+Presents information about deployments in a single view. Contains:
+
+- a table showing the deployments in the system (the left side) The deployments
+  can be further filtered.
+  See [Filtering deployments]({{< relref "#filtering-deployments" >}}).
+- a pane with the deployment details divided into tabs (the right side)
+- a map showing the deployments locations
```

Comment:
Maybe we can add with the same manner info in the brackets where the map is displayed?
Something like "(on the top, opened with **Map** button)". Maybe also info that it is possible to turn it on by default in the widget configuration.

Reply:
Sure thing :+1:

Reply:
Done